qface/idl/domain.py

```diff
@@ -335,7 +335,8 @@ def minorVersion(self):
     @property
     def module_name(self):
         """ returns a capitalized name from the module name """
-        return self.name.split('.')[-1].capitalize()
+        str = module.name.split('.')[-1]
```

Comment:
this means, if capitalisation is required, user needs to change both the templates and the filters that handle property types.
Not a big deal, in ivi we can require that module names should be capitalised in the IDL